src/main/java/org/craftercms/studio/impl/v1/service/content/ContentServiceImpl.java

```diff
@@ -657,6 +661,40 @@ protected String copyContent(String site, String fromPath, String toPath, Set<St
         return retNewFileName;
     }
 
+    protected Map<String, String> getCopyDependencies(String site, String sourceContentPath, String dependencyPath) throws ServiceException {
+        Map<String, String> copyDependency = new HashMap<String, String>();
```

Comment:
Why not just use getItemSpecificDependencies as an implementation of this method?